apis/nucleus/handler/data-property-handler.js

```diff
@@ -256,10 +281,6 @@ class DataPropertyHandler {
     measure.qDef = measure.qDef ?? {};
     measure.qDef.qNumFormat = measure.qDef.qNumFormat ?? {};
 
-    if (isEnabled('MASTER_MEASURE_FORMAT')) {
-      useMasterNumberFormat(measure.qDef);
-    }
-
```

Comment:
why is this removed?

Reply:
but why remove it?
why do we not want to default to use master measure format (like in sense-client)?